app/views/declaration/additional_information.scala.html

```diff
@@ -24,7 +24,7 @@
 
 @this(main_template: views.html.main_template)
```

Comment:
Minor but could we inject it here rather than as an implicit? Then potentially take all `AppConfig` code out of the controllers? (maybe)

Reply:
Yes, you're right. Removed!